services/checks/checks.go

```diff
@@ -248,6 +248,25 @@ func (s *Service) restartChecks(ctx context.Context) {
 	}
 }
 
+// GetSecurityCheckResults returns the results of the STT checks that were run. It returns services.ErrSTTDisabled if STT is disabled.
+func (s *Service) GetSecurityCheckResults() ([]check.Result, error) {
+	settings, err := models.GetSettings(s.db)
+	if err != nil {
+		return nil, err
+	}
+
+	if !settings.SaaS.STTEnabled {
+		return nil, services.ErrSTTDisabled
+	}
+
+	checkResults := make([]check.Result, 0)
+	for _, result := range s.alertsRegistry.getCheckResults() {
+		checkResults = append(checkResults, result.result)
+	}
```

Comment:
If it's possible, better to preallocate slices
```suggestion
	results := s.alertsRegistry.getCheckResults()
	checkResults := make([]check.Result, 0, len(results))
	for _, result := range results {
		checkResults = append(checkResults, result.result)
	}
```

Reply:
done @artemgavrilov